src/typescript-fetch/authenticationMiddleware.ts

```diff
@@ -19,13 +24,10 @@ export class OhipCredentialsProvider {
   appKey: string;
   credentials: OhipCredential[];
   access_token?: string;
-  expiry?: number;
```

Comment:
expiry is now derived from JWT. it's epoch time which can be more accurate than doing `new Date() + expiresIn`. this also simplify things as we don't have to keep updating both access_token and expiry.